intel-retail/automated-self-checkout
# docs: fix the env documentation formatting issue

Fixes: #260

The real fix is on this documentation source md file: **docs_src/environment_variables.md**  
others are by-products of generated docs from `make build-docs` 

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**260**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- run `make build-docs`
- to verify, cd to the directory of containing environment variables docs: `cd docs`
- copy the result of `pwd` command from the console: `pwd`
- open your browser and open this file locally: on the browser address bar put the file protocol and the result of `pwd` with the environment_variables.html file name. so something like this: `file:///home/jimwang/go/src/github.com/intel-retail/automated-self-checkout/vision-self-checkout/docs/environment_variables.html` and you should be able to see now the formatting issue is fixed in the browser html page.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
